lib/gitlab_shell.rb

```diff
@@ -119,7 +119,12 @@ def process_cmd
 
   # This method is not covered by Rspec because it ends the current Ruby process.
   def exec_cmd(*args)
-    Kernel::exec({ 'PATH' => ENV['PATH'], 'LD_LIBRARY_PATH' => ENV['LD_LIBRARY_PATH'], 'GL_ID' => @key_id }, *args, unsetenv_others: true)
+    Kernel::exec({
+      'PATH' => ENV['PATH'],
+      'LD_LIBRARY_PATH' => ENV['LD_LIBRARY_PATH'],
+      'LANG' => ENV['LANG'],
+      'GL_ID' => @key_id
+    }, *args, unsetenv_others: true)
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.